.changeset/techdocs-empty-office.md

```diff
@@ -0,0 +1,8 @@
+---
+'@backstage/techdocs-common': patch
+'@backstage/plugin-techdocs-node': patch
```

Comment:
I think it could be nice to craft a separate changeset for the new packages tbh, without the deprecation label and so on :grin:

Reply:
Gooood call.